examples/complete/variables.tf

```diff
@@ -105,3 +105,9 @@ variable "topics" {
     }
   ]
 }
+
+variable "metrics" {
```

Comment:
please don'e expose this as a variable in the example. We don't wan't to make examples flexible solutions for people to use. Instead I suggest you hard code `metrics` in the example main.tf and add all the allowed values so our our tests test the feature

Reply:
`examples/complete` is using the default standard plan, and the enhanced metrics are only available for enterprise (https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-metrics). However I will hard-code `metrics: []` in the main.tf and remove it from the variables.tf.

Reply:
Editing examples/complete/variables.tf, I am now confused. There are variables for many other module properties (resource tags, access tags, schemas, topics) which are used in main.tf. Why would metrics be treated differently?

Reply:
ah they should not be exposed in the example. I'll create a separate issue for the core team to clean that up, We don't want people to start consuming the examples, as they not intended to be consumable solutions - just reference on how the module can be used.

Reply:
created tracking issue https://github.com/terraform-ibm-modules/terraform-ibm-event-streams/issues/331